content/docs-community/concepts/entries/terms/term-entries/term-entries.md

```diff
@@ -0,0 +1,17 @@
+---
+Title: 'Term Entries'
+Description: 'Refers to a piece of technical content that falls under a given concept, using examples.'
+Subjects:
+  - 'Code Foundations'
+Tags:
+  - 'Documentation'
+  - 'Git'
+  - 'GitHub'
+CatalogContent:
+  - 'introduction-to-open-source'
+  - 'paths/code-foundations'
+---
+
+**Term entries** refer to a piece of technical content that falls under a given concept, using examples. A common example of a term entry is one that describes a [function](https://www.codecademy.com/resources/docs/general/function) or [method](https://www.codecademy.com/resources/docs/general/method) for a given [data type](https://www.codecademy.com/resources/docs/general/data-types) or structure under a specific programming language. A list of existing terms for a given concept is usually located towards the bottom of a concept entry.
```

Comment:
Is it common enough for data types to have functions and methods for them to be called out here?

Reply:
I also mentioned "data structure" to capture as much of what is typically covered as a term entry. I could rephrase to something like the following:

```suggestion
**Term entries** refer to a piece of technical content that falls under a given concept, using examples. A list of existing terms for a given concept is usually located towards the bottom of a concept entry.

Under a specific programming language, terms commonly describe [functions](https://www.codecademy.com/resources/docs/general/function) or [methods](https://www.codecademy.com/resources/docs/general/method) for a given [data type](https://www.codecademy.com/resources/docs/general/data-types) or structure, library, module, etc.
```

Reply:
```suggestion
**Term entries** refer to a piece of technical content that falls under a given concept. These typically include at least one example, and may also include codebyte examples. A list of existing terms for a given concept is usually located towards the bottom of a concept entry.
```

I think this is sufficient without the mention of a specific language's term entry